.editorconfig

```diff
@@ -13,7 +13,7 @@ insert_final_newline = true
 trim_trailing_whitespace = true
 indent_style = tab
 
-[*.yml]
+[*.yml,*.json]
```

Comment:
This is *not* required, core uses _tabs_ for indentation of JSON files

See https://core.trac.wordpress.org/changeset/42460/